R/nested_closeable_modal.R

```diff
@@ -0,0 +1,99 @@
+#' Nested Closeable modal popup
```

Comment:
```suggestion
#' Nested Closeable Modal Popup
```
Use Title Case for titles